### DISCUSSION TITLE
Can we load default data to State bypassing events

### DISCUSSION THREAD

**Discussion by shashikantpawar21** [Eventuous/eventuous#106] [Q&A]
Want to load some initial data in the State without using event ... Is it possible ?

I dont want them to have them in event store also ..That data needs to acts like a default state

**Comment by alexeyzimarev** [comment]
I am not sure what it means. If it's not in event store, where is it?

**Reply by shashikantpawar21** [reply]
I want to have some default records in AggregateState which is not triggered by any api ...
Then on those default records, the user can perform their action which we will add as events in event store.. 

Is it doable ? ..just like a master table in crud ...
